src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const fixedRobot = {};
+
+  for (const key in robot) {
+    if (fixedRobot[robot[key]]) {
```

Comment:
you use robot[key] several times so you should add this in variable

Reply:
2 times i think all right)